config/routes.rb

```diff
@@ -223,6 +223,7 @@
 
   # Help
   get 'help' => 'help#faq', :as => 'help'
+  get 'help/:topic' => 'help#faq'
```

Comment:
@svbergerem, The most recent Hound release includes https://github.com/thoughtbot/hound/commit/93be3aa8dbbc29c8477b7a651799896b4c3bca46 which should let you to disable Ruby reviews. The `.hound.yml` file in master looks right.

Were you able to disable `maxlen` and `quotmarks`?